elfo-core/src/init.rs

```diff
@@ -19,7 +19,6 @@ use crate::{
     context::Context,
     demux::Demux,
     errors::{RequestError, StartError, StartGroupError},
-    memory_tracker::MemoryCheckResult,
```

Comment:
This import fails compilation for non-linux OS. Moved it below under `#[cfg(target_os = "linux")]` section.